.github/workflows/ci-ecr.yaml

```
@@ -82,8 +82,11 @@ jobs:
           ECR_REPOSITORY: ${{ inputs.ecr_repository_name }}
           DOCKER_BUILDKIT: "1"
         run: |
-          IMAGE_TAG=$(git rev-parse HEAD)
-          LATEST_GIT_SHA=$(git ls-remote origin HEAD | cut -f 1)
+          LOCAL_HEAD_SHA="$(git rev-parse HEAD)"
+          REMOTE_HEAD_SHA=$(git ls-remote origin HEAD | cut -f 1)
+
+          IMAGE_TAG="release_${LOCAL_HEAD_SHA}"
```

`-` seems to be much more common than `_` in image tags (e.g. looking at the top few images on DockerHub) so I'd recommend we go with `-`. What do you reckon?

```suggestion
          IMAGE_TAG="release-${LOCAL_HEAD_SHA}"
```

---

nice shout - good to go with conventions!